packages/debug/src/browser/debug-session.tsx

```diff
@@ -521,6 +522,11 @@ export class DebugSession implements CompositeTreeElement {
         if (!terminal) {
             terminal = await this.terminalServer.newTerminal(options);
             await terminal.start();
+            try {
+                await waitForEvent(terminal.onOutput, 1000);
+            } catch (error) {
+                console.warn(`Terminal did not emit output in time, using it anyway: ${error}`);
+            }
```

Comment:
What is the point of waiting for the output if we continue anyway? This could use a comment if there is a difference. Also where do the 1000 ms come from?

Reply:
Nitpick: I believe we do not delay the creation of the terminal, right? We delay opening it.